ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -263,6 +263,23 @@ echo "$(date) - waiting for clusteroperators to finish progressing..."
 oc wait clusteroperators --all --for=condition=Progressing=false --timeout=10m
 echo "$(date) - all clusteroperators are done progressing."
 
+# wait up to 10m for the number of nodes to match the number of machines
+echo "$(date) - waiting for number of nodes to equal number of machines..."
+MACHINECOUNT=$(kubectl get machines -A --no-headers | wc -l)
+NODECOUNT=$(kubectl get nodes --no-headers | wc -l)
+i=0
+until [ $MACHINECOUNT -eq $NODECOUNT ] || [ $i -gt 4 ]
+do
+  echo "$MACHINECOUNT Machines - $NODECOUNT Nodes"
```

Comment:
how do nodecount and machinecount get updated inside of this loop?

Reply:
They don't. :( Fixed now.